index.html

```diff
@@ -432,13 +437,13 @@ <h2>Animals by Type</h2>
           <option value="rabbit">rabbit</option>
           <option value="duck">duck</option>
         </select>
-
         <button class="searchBtn" onclick="fetchHuntingType()">Search</button>
```

Comment:
don't use inline javascript events, try to handle all the JS in the app.js file 
TIP: if you cannot target elements from your js file try to locate your script at the very bottom of your body tag so the HTML can load before the JS 

**Do this with all the inline JS**